winconfig.h

```diff
@@ -1,3 +1,5 @@
+#pragma once
```

Comment:
This file will be used by gcc if we run make again after building with visual studio. I believe this #pragma is supported by gcc, but might need verification (or moved in under the msvc specific part of the file).

Reply:
Just verified on local build, seems to be fine under gcc as well.